radar-upload-frontend/src/axios/file.js

```diff
@@ -107,14 +107,13 @@ export default {
         .reverse());
   },
 
-  async download({ recordId, fileName }) {
-    const url = `${baseURL}/records/${recordId}/contents/${fileName}`;
-    await axios.get(url);
-    downLoadFile(fileName, url);
+  async download({ recordId, fileName, url }) {
+    const fileUrl = url || `${baseURL}/records/${recordId}/contents/${fileName}`;
+    await axios.get(fileUrl);
```

Comment:
I think call get method to download API still loads the file into the memory before being able to save it

Reply:
Fixed in bc6bfb2.

Reply:
> The file name in the download attribute will be the same as the file name on the server. So it will not be a random name, it will be the name that the user (and then the backend) assigned to it.

if that's the case then it's ok.